website/content/docs/configuration/client.mdx

```diff
@@ -382,6 +382,10 @@ see the [drivers documentation](/docs/drivers).
   isolation should be disabled for artifact downloads. Applies only to systems
   where filesystem isolation via [landlock] is possible (Linux kernel 5.13+).
 
+- `set_environment_variables` `(string:"")` - Specifies a comma separated list
+  of environment variables that should be inherited by the artifact sandbox from
+  the Nomad client's environment.
```

Comment:
Do we want to call out the default variables here so that people don't think they need to add `PATH`?